src/plugins/intel_npu/src/plugin/npuw/partitioning/online/snapshot.cpp

```diff
@@ -45,11 +45,35 @@ bool isOp(const std::shared_ptr<ov::Node>& node) {
     }
     return true;
 }
+
+std::vector<ov::element::Type> getWeightsPrecision(const std::shared_ptr<ov::Node>& node) {
```

Comment:
To avoid confusion I'd call it getConstsPrecision - not all consts are actually weights

Reply:
Done